src/plugin/ipc/file/fileconnection.cpp

```diff
@@ -518,38 +518,6 @@ void FileConnection::doLocking()
   _ckpted_file = false;
 }
 
-void FileConnection::handleUnlinkedFile()
-{
-  if ((!jalib::Filesystem::FileExists(_path) && !_isBlacklistedFile(_path)) ||
-      _type == FILE_DELETED ||
-      Util::strStartsWith(jalib::Filesystem::BaseName(_path), ".nfs")) {
-    /* File not present in Filesystem.
-     * /proc/self/fd lists filename of unlink()ed files as:
-     *   "<original_file_name>(deleted)"
-     */
-    string currPath = jalib::Filesystem::GetDeviceName(_fds[0]);
-
-    if (Util::strEndsWith(currPath, DELETED_FILE_SUFFIX)) {
-      JTRACE("Deleted file") (_path);
-      _type = FILE_DELETED;
```

Comment:
We just use a better way (`fstat()`) to detect "unlinked" files (see https://github.com/dmtcp/dmtcp/pull/141/files#diff-5d0e9b6251ae20cbd6e6a83177a3d586R550).